Blog_part_1.md

```diff
@@ -0,0 +1,359 @@
+Load Testing with Locust
+========================
+
+We believe it's critical to test our systems under load to attempt to understand the impact of system traffic. Not only the increase traffic but also the different styles of traffic. From bursty where a wall of traffic comes within a few minutes or even seconds to traffic that increases more uniformly. 
```

Comment:
s/the increase/increased
s/the different styles/different styles
s/bursty/"bursty,"
delete "or even seconds"
s/more uniformly/uniformly